doc/nrf/links.txt

```diff
@@ -564,6 +564,7 @@
 .. _`nRF Sniffer for Bluetooth LE`: https://docs.nordicsemi.com/bundle/nrfutil/page/nrfutil-ble-sniffer/guides/overview.html
 .. _`nRF Connect Bluetooth Low Energy`: https://docs.nordicsemi.com/bundle/nrf-connect-ble/page/index.html
 .. _`Cellular Monitor`: https://docs.nordicsemi.com/bundle/nrf-connect-cellularmonitor/page/index.html
+.. _`Quick Start`: https://docs.nordicsemi.com/bundle/nrf-connect-quickstart/page/index.html
```

Comment:
I don't think we need this link, as this page is not providing any valuable information about the app that isn't already in nRF Connect for Desktop. But if you want to link to it, then please update other mentions of Quick Start in the docs with the link.

Reply:
The reason to provide the link was because the page also shows the supported devices (The GSG page mentions nRF91x1).
If you feel this is still not relevant, I can remove the link.

Reply:
Right. Let's keep it then.